apps/site/lib/site_web/router.ex

```diff
@@ -51,6 +51,13 @@ defmodule SiteWeb.Router do
     get("/_health", HealthController, :index)
   end
 
+  scope "/cms", SiteWeb do
```

Comment:
We have been trying to put new data endpoints behind the `/api` path for organization purposes.

Reply:
I didn't do that because we're using basic auth and this route isn't for API consumers.